pages/common/odps-auth.md

```diff
@@ -0,0 +1,27 @@
+# odps auth
+
+> Manage authorities in ODPS (Open Data Processing Service).
+
+- Show authorities of a user:
+
+`show grants for {{user_name}};`
+
+- Add user to current project:
+
+`add user {{user_name}};`
+
+- Grant a set of authorities to a user:
+
+`grant {{action_list}} on {{object}} {{object_name}} to user {{user_name}};`
+
+- Create a role:
+
+`create role {{role_name}};`
+
+- Grant a set of authorities to a role:
```

Comment:
Perhaps "to a user role" would make this clearer -- let me know if that would be a technically correct way to phrase it.